src/map/modifier.h

```diff
@@ -728,8 +729,8 @@ enum class Mod
     // The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
     // 570 through 825 used by WS DMG mods these are not spares.
     // SPARE = 911, // stuff
```

Comment:
is 911 still a spare? Why jump to 912 if that's the case?

Reply:
You might have a pr in the works that is using 911.

Reply:
Even if I did, if yours merges first, then I would merge master and update. Not leave a gap.